pkg/ingress/secret.go

```diff
@@ -184,9 +184,9 @@ func (c *secretController) onAdd(obj interface{}) {
 		return
 	}
 
-	log.Debugw("secret add event arrived",
-		zap.Any("object", obj),
-	)
+	//log.Debugw("secret add event arrived",
+	//	zap.Any("object", obj),
```

Comment:
It seems that `zap.Any` outputs the `cert` and `pkey` in `[]bytes` and it's not human-friendly. I think we can just print the `name` and `namespace` of this secret, and do not print the body.

Reply:
Yes, can not print.

Reply:
if log "object" with `zap.Any`, got error ` bad access: nil dereference`.
I changed logging field.